entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -9,99 +9,222 @@ uid: core/get-started/aspnetcore/new-db
+You can follow the tutorial by using Visual Studio 2017 on Windows, or by using the .NET Core CLI on Windows, macOS, or Linux.
+
+View this article's sample on GitHub:
+* [Visual Studio 2017 with SQL Server](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb)
+* [.NET Core CLI with SQLite](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCoreNewDb.Sqlite/EFGetStarted.AspNetCore.NewDb).
 
 ## Prerequisites
 
 Install the following software:
 
-* [Visual Studio 2017 15.7](https://www.visualstudio.com/downloads/) with these workloads:
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.7 or later](https://www.visualstudio.com/downloads/) with these workloads:
   * **ASP.NET and web development** (under **Web & Cloud**)
   * **.NET Core cross-platform development** (under **Other Toolsets**)
 * [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
 
-## Create a new project in Visual Studio 2017
+# [.NET Core CLI](#tab/netcore-cli)
+
+* [.NET Core 2.1 SDK](https://www.microsoft.com/net/download/core).
+
+---
+
+## Create a new project
+
+# [Visual Studio](#tab/visual-studio)
 
 * Open Visual Studio 2017
 * **File > New > Project**
-* From the left menu select **Installed > Visual C# > .NET Core**.
+* From the left menu, select **Installed > Visual C# > .NET Core**.
 * Select **ASP.NET Core Web Application**.
 * Enter **EFGetStarted.AspNetCore.NewDb** for the name and click **OK**.
 * In the **New ASP.NET Core Web Application** dialog:
-  * Ensure the options **.NET Core** and **ASP.NET Core 2.1** are selected in the drop down lists
+  * Make sure that **.NET Core** and **ASP.NET Core 2.1** are selected in the drop-down lists
   * Select the **Web Application (Model-View-Controller)** project template
-  * Ensure that **Authentication** is set to **No Authentication**
+  * Make sure that **Authentication** is set to **No Authentication**
   * Click **OK**
 
-Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to your project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+Warning: If you use **Individual User Accounts** instead of **None** for **Authentication** then an Entity Framework Core model will be added to the project in `Models\IdentityModel.cs`. Using the techniques you learn in this tutorial, you can choose to add a second model, or extend this existing model to contain your entity classes.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* Run the following command to create an MVC project:
+
+   ```cli
+   dotnet new mvc -o EFGetStarted.AspNetCore.NewDb
```

Comment:
Nit: I personally think that -n is a bit more intuitive, but you may disagree. There isn't much of a difference on what they do unless both are used.

Reply:
Also explicitly change the current directory so that subsequent usage of 'dotnet' goes against the right directory. See https://github.com/aspnet/EntityFramework.Docs/pull/964.